extensions/ql-vscode/src/extension.ts

```diff
@@ -234,7 +235,10 @@ export async function activate(
   const distributionConfigListener = new DistributionConfigListener();
   await initializeLogging(ctx);
   await initializeTelemetry(extension, ctx);
-  addUnhandledRejectionListener();
+  if (!isCodespacesTemplate()) {
+    addUnhandledRejectionListener();
+  }
+
```

Comment:
I'm concerned about just disabling the unhandled rejection listener for the template. It means that none of our new users will get the benefits of it. I think we'd be better off inspecting the stack frames for references to our sources (as we discussed on our chat). 

My biggest concern is that we're going to display errors from other extensions (I think it's even possible for this to happen now, just no one has reported it yet).